docs/application/native/guides/machine-learning/machine-learning-service.md

```diff
@@ -23,47 +29,167 @@ Follow the steps below to enable your application to use Machine Learning Servic
 
-An ML developer adds a pipeline description which includes preprocessing and invoking a neural network model, and an application developer can get the registered pipeline.
+The following code snippet shows an example of how the Machine Learning Service API can be used in this scenario. Note that checking return value and error handling is omitted for brevity.
+
+### AI developers launching the object detection ML service
+
+AI developers can manage the pipeline description.
+
+1. Set the pipeline description with a service name "object_detection_service":
 
-1. Set the pipeline description:
     ```c
-    const gchar my_pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! tensor_sink async=false";
-    int status;
-
-    status = ml_service_set_pipeline ("my_pipeline", my_pipeline);
-    if (status != ML_ERROR_NONE) {
-      // handle error case
-      goto error;
-    }
+    const gchar object_detection_pipeline[] = "\
+    tensor_query_serversrc port=#reserved_port_number host=localhost ! \
+    other/tensors,num_tensors=1,types=uint8,format=static,dimensions=3:320:320:1,framerate=0/1 ! \
+    tensor_transform mode=arithmetic option=typecast:float32,add:0.0,div:255.0 ! \
+    queue ! tensor_filter framework=tensorflow2-lite model=/path/to/yolov5s/model/file.tflite ! \
+    other/tensors,num_tensors=1,types=float32,format=static,dimensions=85:6300:1:1,framerate=0/1 ! \
+    tensor_decoder mode=bounding_boxes option1=yolov5 option2=/path/to/coco/label.txt option4=320:320 option5=320:320 ! \
+    video/x-raw,format=RGBA,width=320,height=320 ! tensor_converter ! \
+    other/tensors,format=static,num_tensors=1,dimensions=4:320:320:1,types=uint8 ! \
+    tensor_query_serversink async=false sync=false";
+
+    // Set the pipeline with service name.
+    ml_service_set_pipeline ("object_detection_service", object_detection_pipeline);
     ```
 
-2. Get the pipeline description and construct this for various data-streamed services:
+2. Get or delete the pipeline description of the service name:
+
     ```c
+    // Get the pipeline description of given service name.
     gchar *pipeline;
-    int status;
-    ml_pipeline_h handle;
-
-    status = ml_service_get_pipeline ("my_pipeline", &pipeline);
-    if (status != ML_ERROR_NONE) {
-      // handle error case
-      goto error;
-    }
-
-    status = ml_pipeline_construct (pipeline, NULL, NULL, &handle);
-    if (status != ML_ERROR_NONE) {
-      // handle error case
-      goto error;
-    }
-
-    // run the service.
-
-    error:
-    ml_pipeline_destroy (handle);
-    g_free (pipeline);
+    ml_service_get_pipeline ("object_detection_service", &pipeline);
+
+    // Delete the pipeline description of given service.
+    ml_service_delete_pipeline ("object_detection_service");
+    ```
+
+AI developers can manage the ML service.
```

Comment:
```suggestion
AI developers can manage the ML service by doing the following:
```
Introduce a list with a complete sentence, not a partial one that's completed by the list items.
Reference: https://developers.google.com/style/lists#introductory-sentences-for-lists